src/App/App.js

```diff
@@ -10,25 +11,38 @@ const beValues = [
   {
     id: '0',
     caption: 'Card one header',
-    content: loremIpsum
+    content: loremIpsum,
+    selected: true
   }, {
     id: '1',
     caption: 'Card two header',
-    content: loremIpsum
+    content: loremIpsum,
+    selected: true
   }, {
     id: '2',
     caption: 'Card three header',
-    content: loremIpsum
+    content: loremIpsum,
+    selected: false
   }
 ];
 
 
 function App() {
-  const cards = beValues.map(one =>
+  const [cardsState, setCardsState] = useState(beValues);
+
+  const cardsRendered = cardsState.map(one =>
     <Card
       key={one.id}
       caption={one.caption}
       content={one.content}
+      selected={one.selected}
+      onSelect={() => setCardsState(
+        [...cardsState].map(newOne =>
```

Comment:
I think cardsState without quad brackets will be enough.
Correct it and merge plz.